.drone.yml

```diff
@@ -263,7 +263,7 @@ platform:
 
 steps:
 - name: build
-  image: rancher/dapper:v0.5.0
+  image: rancher/dapper:v0.6.0
```

Comment:
I think I mentioned it on slack, but newer versions of dapper crash on arm, as seen in the CI logs. Don't bump this here.